packages/site/landing/index.html

```diff
@@ -62,16 +62,19 @@
                 </div>
             </div>
         </div>
-        <svg class="circle-wrapper" id="circles" viewBox="0 0 1550 1550" width="1300" height="1300" stroke-width="50"
-            fill="transparent" stroke-linecap="round">
-            <circle cx="775" cy="775" r="750" />
-            <circle cx="775" cy="775" r="700" />
-            <circle cx="775" cy="775" r="650" />
-            <circle cx="775" cy="775" r="600" />
-            <circle cx="775" cy="775" r="550" />
-            <circle cx="775" cy="775" r="500" />
-            <circle cx="775" cy="775" r="450" />
-        </svg>
+        <nimble-theme-provider theme="dark">
```

Comment:
Traces in dark theme
![image](https://github.com/ni/nimble/assets/1588923/8475cc8c-d5c8-41c1-a88c-3b7f2c0cb937)

Traces in light theme:
![image](https://github.com/ni/nimble/assets/1588923/a8b7c7c7-1edd-4cf0-9c88-d8f89d00186d)

Reply:
@NIbokeefe, wanna weigh in on whether you like this direction? I think it's technically cool to use our official graph tokens for the home page rings but if you prefer the aesthetic of the original design then I'd say it's your call.

Reply:
@jattasNI @rajsite 

The dark themed traces look interesting because they are softer against the white.  The light theme looks too harsh.  Graph colors were designed to be viewed at very thin sizes, it doesn't hold up for these thick traces.

I still probably prefer the original NIBrand colors, but again the dark graph colors are #2 for me.  What prompted the possible changes?

Reply:
Just seemed neat to use the same colors someone would see on a graph / elsewhere in the design system. No real reason to change. 

Keeping the current brand colors is 100% Gucci. One question is would we use the same exact set of colors for an actual dark mode landing page (ie dark background etc)?

Reply:
Gotcha, thanks!

Yes, I tested all the colors on dark mode and it looks great.   Here's what that looks like:

<img width="1091" alt="Screenshot 2024-06-27 at 11 17 53 AM" src="https://github.com/ni/nimble/assets/106594349/055f63be-5afd-48a0-9fa6-2f199e361a28">

Reply:
Love it, let's keep it as-is!